packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -119,11 +124,14 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
           return null
         }
 
-        const issueRes = await integrationManager.getIssues()
+        const projectKeys = (projectKeyFilters ?? []).map((projectKeyFilter) => IntegrationRepoId.split(projectKeyFilter).projectKey!)
+
+        const issueRes = await integrationManager.getIssues(queryString, isJQL, projectKeys)
 
         if (issueRes instanceof Error) {
-          sendToSentry(issueRes, {userId, tags: {teamId}})
-          return null
+          return connectionFromTasks([], first, {
```

Comment:
Why return an empty connection out of interest instead of null & sending the error to Sentry?

Also, it's not from this PR, but I noticed that we're checking is `provider` exists above, but we're using `loadNonNull` so it's guaranteed that it won't be nullable

Reply:
@nickoferrall As here we are returning an error that should be displayed in the UI
![image](https://user-images.githubusercontent.com/466991/164472163-08d9d39c-4d07-4ce1-ac27-7345fa37ae7e.png)

Reply:
Removed provider check